src/main/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoader.groovy

```diff
@@ -448,6 +450,12 @@ class DockerThreadContextClassLoader implements ThreadContextClassLoader {
         constructor.newInstance()
     }
 
+    public Object createExecCallback(OutputStream out, OutputStream in) {
```

Comment:
You need to define this within `ThreadContextClassLoader`. Also, return 'def' instead of 'Object'.